auth/user_mgt_test.go

```diff
@@ -49,9 +50,11 @@ var testUser = &UserRecord{
 			DisplayName: "Test User",
 			PhotoURL:    "http://www.example.com/testuser/photo.png",
 			Email:       "testuser@example.com",
+			UID:         "testuser@example.com",
```

Comment:
What is the intention here? Setting UID to email (and phone number below) looks unusual.